Dockerfile.test.rhtap

```diff
@@ -0,0 +1,24 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.21 AS builder
```

Comment:
is brew the right registry? I know homebrew is a mac-os thing, so I'm just double checking